cpp/tests/groupby/common/groupby_test_util.hpp

```diff
@@ -45,11 +48,25 @@ inline void test_single_agg(column_view const& keys,
     requests[0].aggregations.push_back(std::move(agg));
 
     experimental::groupby::groupby gb_obj(table_view({keys}),
-        ignore_null_keys, keys_are_sorted, column_order, null_precedence);
+        include_null_keys, keys_are_sorted, column_order, null_precedence);
 
     auto result = gb_obj.aggregate(requests);
-    expect_tables_equal(table_view({expect_keys}), result.first->view());
-    expect_columns_equal(expect_vals, result.second[0].results[0]->view(), true);
+
+    if (stable_order) 
+    {
+        expect_tables_equal(table_view({expect_keys}), result.first->view());
+        expect_columns_equal(expect_vals, *result.second[0].results[0], true);
+    } else {
+        auto const sort_order = experimental::sorted_order(result.first->view(),
+            {}, {null_order::AFTER});
+        auto const sorted_keys = experimental::gather(result.first->view(),
+            *sort_order);
+        auto const sorted_vals = experimental::gather(
+            table_view({result.second[0].results[0]->view()}), *sort_order);
```

Comment:
I rarely comment on formatting, as `clang-format` will fix most things (including this I believe) but these will all be within the column limit of 120 or 140 (whichever we decide) and not need to line break here, it will look much cleaner in 3 lines instead of 6.

Reply:
But 80 chars allows 3 editors side by side.

Reply:
You can leave it for now if you want, but `clang-format` will change it in the future :)

Reply:
Wait wait wait. Can there be a clang format rule to shorten lines above 120 chars but leave them alone if they already are? 🔱

Reply:
That would defeat the purpose of having clang-format enforce a consistent formatting style.

Reply:
Is it too late to argue in favour of 80 character limits?

Reply:
There was agreement for 120 or 140. Imo, being able to have 3 editors side-by-side is not a motivating reason for 80 char limit.

Reply:
Nice bike shed you built here.

Reply:
3 editors is a luxury that I rarely use. But 2 editors is a necessity to reduce visual context switch which some devs are more sensitive to than others. I got myself a 2560x1440 monitor which can do 2x120 but Nvidia's standard issue is 1920x1200 which won't.

But if advocating for 80 is a lost cause, can I vote for 120 rather than 140?